docs/components/Dropdown.vue

```diff
@@ -7,15 +7,15 @@
     <div slot="example">
       <div class="component-example">
         <ao-dropdown :show-dropdown="showDropdown">
-          <a
+          <ao-dropdown-item
             v-for="(link, index) in links"
             slot="dropdown-items"
             :key="index"
             :href="link.url"
-            class="ao-dropdown__link"
+            class="ao-dropdown__item"
```

Comment:
i dont think the class is needed please remove as it may become confusing if it is useless